webtorrent/transport.go

```diff
@@ -49,11 +49,17 @@ func (me *wrappedPeerConnection) Close() error {
 	return err
 }
 
-func newPeerConnection(logger log.Logger) (*wrappedPeerConnection, error) {
+func newPeerConnection(logger log.Logger, iceServers []string) (*wrappedPeerConnection, error) {
 	newPeerConnectionMu.Lock()
 	defer newPeerConnectionMu.Unlock()
 	ctx, span := otel.Tracer(tracerName).Start(context.Background(), "PeerConnection")
-	pc, err := api.NewPeerConnection(config)
+
+	pcConfig := config
+	if iceServers != nil {
```

Comment:
Is this check necessary? Can you just have a default value in the ClientConfig, or will pion treat nil as the default anyway?

Reply:
This was in fact an unnecessary check. If Pion receives `nil` or an empty list of ICE Servers, it will simply ignore it.